README.md

```diff
@@ -118,6 +118,35 @@ Available files, and their formats:
      * One line per input read with taxonomic classification and detailed hit
        information.
 
+* `samplereads/`: Itermediate, example reads by taxonomic category
+  * ex: `SRR14530724.sr.tsv.gz`
+  * TSV
+  * 100,000 examples (or as many as available) from each sample of:
```

Comment:
Mention how these were sampled? If I understand the code correctly it takes the first `n` reads instead of eg random sampling. 

"First 100,000 examples (or as many as available)...."

Reply:
Good point!  Added in 80db304